kinde-core/kinde-accounts-api.yaml

```
@@ -0,0 +1,531 @@
+      properties:
+        data:
+          $ref: '#/components/schemas/UserProfile'
+      required:
+        - data
+    CurrentOrganizationResponse:
+      type: object
+      properties:
+        data:
+          $ref: '#/components/schemas/Organization'
+      required:
+        - data 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)